python/pip_install/tools/wheel_installer/BUILD.bazel

```diff
@@ -0,0 +1,68 @@
+load("@rules_python//python:defs.bzl", "py_binary", "py_library", "py_test")
+load("//python/pip_install:repositories.bzl", "requirement")
+
+py_library(
+    name = "lib",
+    srcs = [
+        "namespace_pkgs.py",
+        "wheel.py",
+        "wheel_installer.py",
+    ],
+    deps = [
+        "//python/pip_install/tools/lib",
+        requirement("installer"),
+        requirement("pip"),
+        requirement("setuptools"),
+    ],
+)
+
+py_binary(
+    name = "wheel_installer",
+    srcs = [
+        "wheel_installer.py",
+    ],
+    deps = [":lib"],
+)
+
+py_test(
+    name = "namespace_pkgs_test",
+    size = "small",
+    srcs = [
+        "namespace_pkgs_test.py",
+    ],
+    tags = ["unit"],
```

Comment:
Is this tags=unit used by something elsewhere (didn't see it)? I'd think that size=small was sufficient info to filter (small usually means unit test)

Reply:
I think it's probably unused. I saw a few `test_tag_filters` to exclude integration test tags. I'll try remove it and see.